src/Resources/config/form.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+        http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="algatux_influx_db.form.type" class="Algatux\InfluxDbBundle\Form\Type\AbstractInfluxChoiceType"
+                 abstract="true">
+            <call method="setConnectionRegistry">
```

Comment:
This is an abstract class. Use may extends it and pass other arguments onto the constructor.

Using a called setter ensure the service passing will be never broken. :-)

BTW, with this way, user don't have to take care about parent constructor arguments at all.